src/test/java/seedu/address/model/timetable/time/util/TimeUtilsTest.java

```diff
@@ -0,0 +1,364 @@
+package seedu.address.model.timetable.time.util;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.model.timetable.util.TypicalTime.EIGHT_AM;
+import static seedu.address.model.timetable.util.TypicalTime.EIGHT_PM;
+import static seedu.address.model.timetable.util.TypicalTime.ELEVEN_AM;
+import static seedu.address.model.timetable.util.TypicalTime.FIVE_PM;
+import static seedu.address.model.timetable.util.TypicalTime.FOUR_PM;
+import static seedu.address.model.timetable.util.TypicalTime.NINE_AM;
+import static seedu.address.model.timetable.util.TypicalTime.NINE_PM;
+import static seedu.address.model.timetable.util.TypicalTime.ONE_PM;
+import static seedu.address.model.timetable.util.TypicalTime.SEVEN_PM;
+import static seedu.address.model.timetable.util.TypicalTime.SIX_PM;
+import static seedu.address.model.timetable.util.TypicalTime.TEN_AM;
+import static seedu.address.model.timetable.util.TypicalTime.TEN_PM;
+import static seedu.address.model.timetable.util.TypicalTime.THREE_PM;
+import static seedu.address.model.timetable.util.TypicalTime.TWELVE_PM;
+import static seedu.address.model.timetable.util.TypicalTime.TWO_PM;
+import static seedu.address.model.timetable.util.TypicalTimetable.FULL_CONFLICT_TIMETABLE_A;
+import static seedu.address.model.timetable.util.TypicalTimetable.FULL_CONFLICT_TIMETABLE_B;
+import static seedu.address.model.timetable.util.TypicalTimetable.TIMETABLE_A;
+import static seedu.address.model.timetable.util.TypicalTimetable.TIMETABLE_B;
+import static seedu.address.model.timetable.util.TypicalTimetable.TIMETABLE_C;
+import static seedu.address.model.timetable.util.TypicalTimetable.TIMETABLE_D;
+import static seedu.address.model.timetable.util.TypicalTimetable.TIMETABLE_E;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.timetable.Timetable;
+import seedu.address.model.timetable.time.SchoolDay;
+import seedu.address.model.timetable.time.TimeBlock;
+import seedu.address.model.timetable.time.TimePeriod;
+import seedu.address.model.timetable.time.TimeSlot;
+
+
+class TimeUtilsTest {
+
+    @Test
+    public void getCommonIntervals_twoTimetablesMonday_oneTimeBlock() {
+        List<Timetable> timetables = List.of(TIMETABLE_A, TIMETABLE_B);
```

Comment:
Is the list here changed in anyway due to your tests below? If not, consider shifting this to a global variable since there are a lot of tests using the same list below. The same goes to the other lists below too.

Reply:
Ok